### PR TITLE
Sponsor level list

### PR DESCRIPTION
**Pull request by sesas**
Cleaned up PR from: https://github.com/pybay/pybay/pull/113 
PR from JD for speakers_list.html and sponsor_level_list.html templates
"speakers list needs new work per GL work but follows the Variant wireframes. sponsor level list is ok per GL."

### CODE REVIEW

**Comment by sesas** [comment]
@epsy Fixed the conflict

**Comment by superglu** [comment]
Apparently I don't have push access to this. Due to PR #123, please remove any changes to speaker_list.html to avoid conflicting with that PR.

**Comment by sesas** [comment]
@superglu I reverted the changes to `speaker_list.html` and the only file remaining is `sponsor_level_list.html`. Would you be able to review this PR and merge?

File: pybay/templates/frontend/sponsor_level_list.html [hunk 1]
```diff
@@ -15,15 +15,26 @@ <h1 class="large-h1">{{ title }}</h1>
     <section class="sponsor-list">
       {% for sponsor in sponsors %}
         <div class="sl-sponsor">
+          <!-- move this section into sl-blurb --
           <a class="sl-logo" href="{{ sponsor.external_url }}">
```
Comment (epsy) [inline pybay/templates/frontend/sponsor_level_list.html:19]: please remove commented code

File: pybay/templates/frontend/sponsor_level_list.html [hunk 2]
```diff
@@ -15,15 +15,26 @@ <h1 class="large-h1">{{ title }}</h1>
...
             <img
               alt="{{ sponsor.name }}"
               src="{{ sponsor.website_logo.url }}"
             />
           </a>
+          -- end move -->
           <div class="sl-blurb">
             <h3><a href="{{ sponsor.external_url }}">{{ sponsor.name }}</a></h3>
+            <!-- replace link with logo --
             <p class="sl-link"><a href="{{ sponsor.external_url }}">{{ sponsor.external_url }}</a></p>
```
Comment (epsy) [inline pybay/templates/frontend/sponsor_level_list.html:29]: same

**Review by epsy — CHANGES_REQUESTED** [review]
Please rebase

**Pull request closed without merging**